Cargo.toml

```diff
@@ -80,11 +76,8 @@ members = [
 ]
 
 [workspace.dependencies]
-alloy-consensus = "0.4.2"
-alloy-eips = "0.4.2"
-alloy-primitives = { version ="0.8.7" , features = ["map-hashbrown"] }
+alloy = { version = "0.4.2", default-features = false, features = ["std"] }
 alloy-rlp = "0.3.8"
```

Comment:
Should `alloy-rlp` get bumped to "0.4.2" also?

Reply:
The latest version of that one is 0.3.8 (and other 0.4.2 crates do depend on this one, so no duplicates).